frontend/src/scenes/insights/Insight.tsx

```diff
@@ -121,7 +134,47 @@ export function Insight({ insightId }: { insightId: InsightShortId | 'new' }): J
                     />
                 }
                 buttons={
-                    <div className="insights-tab-actions">
+                    <div className="insights-tab-actions space-x-05">
+                        {insightMode === ItemMode.View && (
+                            <More
```

Comment:
One quick note I have from looking just at the screenshots is that this could maybe use a separator, as the button is easy to miss here and is a bit different from the rest. See the dashboard `buttons` for a similar situation:
<img width="295" alt="Screen Shot 2022-05-25 at 20 23 58" src="https://user-images.githubusercontent.com/4550621/170339015-a95c6bda-0272-4df1-8324-c29219ae7727.png">